app/konnect/compatibility.md

```diff
@@ -27,6 +27,15 @@ data plane.
 | {{site.ee_product_name}} 2.4.x or earlier | <i class="fa fa-times"></i>    | --
 
 
+## {{site.mesh_product_name}} compatibility
+
+To use [Mesh Manager](/konnect/mesh-manager/), you must also use a compatible version of {{site.mesh_product_name}}. 
+
+|                                  | {{site.konnect_saas}} |
+|--------------------------------|:---------------------:|
+| {{site.mesh_product_name}} 2.3.x | <i class="fa fa-check"></i> |
```

Comment:
❗ Reviewers: Charly said that versions later than 2.2.x would be supported. 2.3.x will be the next minor version, but a patch version could come out for 2.2.x. Is there a better way to represent this than what I did?

Reply:
I would suggest adding a "First supported patch version" column here, same as the table above. For 2.3.x, that might mean 2.3.0, and that's fine. When the supported 2.2.x patch comes out, we can add an entry for it here as well.